provider/provider.go

```diff
@@ -595,10 +590,24 @@ func resourceShorelineObject(configJsStr string, key string) *schema.Resource {
 				return false
 			}
 		}
+
+		// NOTE: This actually messes up the file objects. Need a suppress function that's just for acceptance test comparisions.
+		//notStored, isBool := GetNestedValueOrDefault(attrMap, ToKeyPath("not_stored"), nil).(bool)
```

Comment:
What is this added for?

Reply:
This was an attempt to suppress errors around the `file` object's local filename (which doesn't even go to the backend). Unfortunately it has side-effects that break the `file` object.